packages/lucide-svelte/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "lucide-svelte",
+  "description": "A Lucide icon library package for Svelte applications",
+  "version": "0.16.26-beta.0",
+  "license": "ISC",
+  "homepage": "https://lucide.dev",
+  "bugs": "https://github.com/lucide-icons/lucide/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lucide-icons/lucide.git",
+    "directory": "packages/lucide-svelte"
+  },
+  "author": "Eric Fennis",
+  "amdName": "lucide-svelte",
+  "main": "dist/cjs/lucide-svelte.js",
+  "main:umd": "dist/umd/lucide-svelte.js",
+  "module": "dist/esm/lucide-svelte.js",
```

Comment:
Hey, just tried the ludice-svelte package with SvelteKit and run into some issues. Svelte requires an 
```json
"svelte": "dist/esm/lucide-svelte.js"
```
entry inside the package.json to resolve the source modules correctly.

Reply:
Thanks a lot! I missed this information. I will add this. Thanks for testing!

Let me know if you find more issues